laborator/content/operatii-memorie-gdb/solution/3-pixels/pixel.h

```diff
@@ -0,0 +1,94 @@
+#ifndef __PIXEL_H__
+#define __PIXEL_H__
+
+typedef struct Pixel {
+	unsigned char R;
+	unsigned char G;
+	unsigned char B;
+} Pixel;
+
+typedef struct Picture {
+	int height;
+	int width;
+	Pixel **pix_array;
+} Picture;
+
+Picture *generatePicture(int height, int width, Pixel **pix_array)
+{
+	Picture *pic = malloc(sizeof(*pic));
+
+	pic->height = height;
+	pic->width = width;
+
+	pic->pix_array = malloc(pic->height * sizeof(*(pic->pix_array)));
+
+	for (int i = 0; i < pic->height; ++i)
+		pic->pix_array[i] = malloc(pic->width * sizeof(*(*(pic->pix_array))));
```

Comment:
Add some error handling logic. Also add it to the skeleton: https://github.com/systems-cs-pub-ro/iocla/blob/master/laborator/content/operatii-memorie-gdb/3-pixels/pixel.h.

Reply:
Ditto.